Header header

float64 lat
float64 long

float64 alt

float64 utm_easting
float64 utm_northing
uint16 utm_zone
string utm_letter

int8 fix
